Dockerfile

```
@@ -15,12 +15,12 @@ RUN gem install bundler
 RUN bundle config set deployment 'true'
 RUN bundle config set without 'development test'
 
+WORKDIR /usr/src/app
+
 COPY Gemfile Gemfile.lock ./
 RUN bundle install
 
 COPY package.json yarn.lock ./
-RUN yarn install
-
-WORKDIR /usr/src/app
+RUN yarn install --check-files
```

I don't think we want to check the files in production.

```suggestion
RUN yarn install
```